frontend/app/components/work-packages/wp-display-attr/wp-display-attr.directive.html

```diff
@@ -1,7 +1,7 @@
 <span>
   <span class="wp-table--cell-span inplace-edit--read-value--value __d__cell" >
 
-    <span class="__d__renderer">
+    <span class="__d__renderer wp-table--cell-span--value">
```

Comment:
@HDinger can you please use a different class name. The `wp-table--cell-span` should be remove altogether IMO as the directive is also used outside the context of the wp-table. Additionally, the `wp-table--cell-span` hierarchy is parallel to the `inplace-edit` hierarchy.